assets/aml-benchmark/components/batch-inference-preparer/spec.yaml

```diff
@@ -31,6 +31,10 @@ inputs:
         "_batch_request_metadata": ###<_batch_request_metadata>
       }
     optional: False
+  label_column_name:
+    type: string
+    optional: True
+    description: The key that stores the ground truth value.
```

Comment:
nit: We can update the description as well like you have for `batch-inference-preparer`- "The label column name.".

Reply:
changed